jose-jwt/JWTSettings.cs

```diff
@@ -58,7 +58,7 @@ public JwtSettings()
             { JwsAlgorithm.ES256, new EcdsaUsingSha(256) },
             { JwsAlgorithm.ES384, new EcdsaUsingSha(384) },
             { JwsAlgorithm.ES512, new EcdsaUsingSha(521) }
-#elif NETSTANDARD || NET461 || NET472
+#elif NETSTANDARD || NET461 || NET472 || NET

```

Comment:
as mentioned in `AesGcmNetCore.cs` comment, no need to touch all project with with extra targeting `|| NET` as fix is just checking length and should work just fine on every plarform.